lib/views/index.erb

```diff
@@ -1,30 +1,56 @@
 <% if @my_cart.count != 0 %>
-  <div id="shopping_cart">
-    Shopping cart
+<div class="cell">
```

Comment:
Don't worry about testing this method in the model. If you just introduce this, I can make sure I talk you through the way this should be unit tested.